projects/icons/scripts/process-icons.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line @taiga-ui/no-deep-imports
+import {tuiIsString} from '@taiga-ui/cdk/utils';
```

Comment:
package `@taiga-ui/icons` has no deps.
I think we should exclude this rule for `projects/icons` and this line

Reply:
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/12021443/213391801-196d348d-12e4-4835-aedd-b69bfdf56597.png">